doc/release-notes.md

```diff
@@ -64,7 +64,18 @@ updates. This should be fixed now.
 
 See detailed [set of changes](https://github.com/dashpay/dash/compare/v0.14.0.1...dashpay:v0.14.0.2).
 
-TODO
+- [`d2ff63e8d`](https://github.com/dashpay/dash/commit/d2ff63e8d) Use std::unique_ptr for mnList in CSimplifiedMNList (#3014)
+- [`321bbf5af`](https://github.com/dashpay/dash/commit/321bbf5af) Fix excessive memory use when flushing chainstate and EvoDB (#3008)
+- [`0410259dd`](https://github.com/dashpay/dash/commit/0410259dd) Fix 2 common Travis failures which happen when Travis has network issues (#3003)
+- [`8d763c144`](https://github.com/dashpay/dash/commit/8d763c144) Only load signingActiveQuorumCount + 1 quorums into cache (#3002)
+- [`2dc1b06ec`](https://github.com/dashpay/dash/commit/2dc1b06ec) Remove skipped denom from the list on tx commit (#2997)
+- [`dff2c851d`](https://github.com/dashpay/dash/commit/dff2c851d) Update manpages for 0.14.0.2 (#2999)
+- [`fb6f0e04d`](https://github.com/dashpay/dash/commit/fb6f0e04d) Bump version to 0.14.0.2 and copy release notes (#2991)
```

Comment:
Would prefer to preserve the order they appear in `git log` i.e. this should be the first one (at the bottom of the list)
```suggestion
```

Reply:
Strange, this is the order that I got from `git log --oneline --no-merges v0.14.0.1..v0.14.0.x`

Reply:
Hmm... indeed. That's probably because #2989 wasn't rebased after #2991 was merged. `git log --oneline --graph` shows a bit more correct picture (imo) in terms of what was merged first.

Reply:
Ah ok, thanks. Will use `--graph` in the future :) Force pushed the fixed version